.circleci/config.yml

```diff
@@ -0,0 +1,92 @@
+version: 2
+
+.steps_template: &steps_terraform_static_analysis
+  steps:
+    - checkout
+    - run:
+        name: "Check: Validate tf files (terraform validate)"
+        command: |
+            for num in 1 2; do (
+              cd lesson0${num} &&
+              terraform init -input=false &&
+              terraform validate -json &&
+              cp ../slides/lesson${num}/solution/main.tf main.tf
+              terraform validate -json &&
+              cp -f ../slides/lesson${num}/solution/main_hard_code.tf main.tf
+              terraform validate -json &&
+              cd - &&
+              echo "√ ${REPLY}") || exit 1
+            done
+    - run:
+        name: "Check: Terraform formatting (terraform fmt)"
+        command: |
+            if [ `terraform fmt --list=true -diff=true -write=false | tee format-issues | wc -c` -ne 0 ]; then
+              echo "Some terraform files need be formatted, run 'terraform fmt' to fix"
+              echo "Formatting issues:"
+              cat format-issues
+              exit 1
+            fi
+    - run:
+        name: "Install: tflint"
```

Comment:
Since this may take a while, what are your thoughts on this being another job in circleci?